runtime/isolate_configuration.cc

```diff
@@ -202,12 +203,17 @@ PrepareKernelMappings(const std::vector<std::string>& kernel_pieces_paths,
 std::unique_ptr<IsolateConfiguration> IsolateConfiguration::InferFromSettings(
     const Settings& settings,
     const std::shared_ptr<AssetManager>& asset_manager,
-    const fml::RefPtr<fml::TaskRunner>& io_worker) {
+    const fml::RefPtr<fml::TaskRunner>& io_worker,
+    IsolateLaunchType launch_type) {
```

Comment:
Should this be in Settings?

Reply:
No, it can vary from isolate to isolate.